bin/ck8s

```diff
@@ -5,7 +5,7 @@
 set -e -o pipefail
 
 here="$(dirname "$(readlink -f "$0")")"
-# shellcheck disable=SC1090
+# shellcheck source=bin/common.bash
 source "${here}/common.bash"
 
 usage() {
```

Comment:
Please add the new commands to the list here as well

Reply:
Added completions as well! ;)